modules/exploits/multi/browser/chrome_jscreate_sideeffect.rb

```diff
@@ -0,0 +1,370 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ManualRanking
+
+  include Msf::Post::File
+  include Msf::Exploit::Remote::HttpServer
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Google Chrome 80 JSCreate side-effect type confusion exploit',
+      'Description'    => %q{
+      This module exploits an issue in Chrome 80.0.3987.87 (64 bit) on Windows 10 --no-sandbox.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         => [
+          'saelo', # discovery
+          'István Kurucsai', # exploit
+          'timwr', # metasploit copynpasta
```

Comment:
```suggestion
          'timwr', # metasploit copypasta
```
Looks like you got a bad copypasta when you copypasta'd the `copynpasta` 😜